pkg/webui/oauth/views/login/index.js

```diff
@@ -51,7 +52,10 @@ const validationSchema = Yup.object().shape({
 const appRoot = selectApplicationRootPath()
 
 @withRouter
-@connect()
+@connect(null, {
+  push,
```

Comment:
not used

Reply:
also, this breaks navigation to the register page, because you dont have `dispatch` in `props`. `L84`